src/main/java/redis/embedded/util/RedisExecutor.java

```diff
@@ -0,0 +1,40 @@
+package redis.embedded.util;
+
+import java.util.regex.Pattern;
+
+public class RedisExecutor {
```

Comment:
The `Executor` part of the name make me think of `java.util.concurrent.Executor`, which lead to think it control how the process is run.
Could it be renamed to `RedisExecutable` (or something like that) ?